src/Bicep.Cli.IntegrationTests/ProgramTests.cs

```diff
@@ -204,7 +204,7 @@ public void BuildSingleFileShouldProduceExpectedErrors(DataSet dataSet)
         [DynamicData(nameof(GetInvalidDataSets), DynamicDataSourceType.Method, DynamicDataDisplayNameDeclaringType = typeof(DataSet), DynamicDataDisplayName = nameof(DataSet.GetDisplayName))]
         public void BuildSingleFileToStdOutShouldProduceExpectedErrors(DataSet dataSet)
         {
-            var outputDirectory = dataSet.SaveFilesToTestDirectory(TestContext, dataSet.Name);
```

Comment:
This is going to make debugging these tests after they run very difficult. What was the issue with the original path?

Reply:
My motivations:
* Not having to worry about length/sanitization - for example, I get a lot of 'path too long' errors on Windows when trying to run `git clean -xfd .`
* I've never used knowledge of the test output structure for debugging - I always look at the error message in the failure. It feels more user-friendly to focus on better error messages if there are cases where the errors are not sufficient for debugging.
* Solves the issue with upgrading to MSTest.TestAdapter 2.2.1 (less of a concern)

Reply:
Yeah it's rare to have to look at the file structure. I guess it is less likely to have the need since the file names will be random (no race conditions).

Reply:
BTW, we shoukd separate this change from the nuget upgrade given there are issues with it in VS.

Reply:
Good point, done